app/assets/javascripts/helpers/colorSelector.js

```diff
@@ -0,0 +1,29 @@
+((function (App) {
+  'use strict';
+
+  App.Helper.ColorSelector = {
```

Comment:
I think it's better if you use a Backbone view for this. Have a look at `App.View.FlagColorsView` for example.